### PR TITLE
Use udev to determine if a drive is removable on GNU/Linux

### PR DESCRIPTION
**Pull request by jviotti**

### CODE REVIEW

File: scripts/linux.sh [hunk 1]
```diff
@@ -22,12 +22,12 @@ for disk in $DISKS; do
...
-  removable=`lsblk -d $device --output RM | ignore_first_line | trim`
-
-  if [[ "$removable" == "1" ]]; then
+  eval "`udevadm info --query=property --export --export-prefix=UDEV_ --name=$disk`"
+  if [[ "`lsblk -d $device --output RM | ignore_first_line | trim`" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_MEDIA_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_BUS" == "usb" ]]
+  then
     echo "system: False"
```
Comment (agherzan) [inline scripts/linux.sh]: This error seems way to generic. Can you make it more detailed / descriptive? Same for True.

File: scripts/linux.sh [hunk 2]
```diff
@@ -22,12 +22,12 @@ for disk in $DISKS; do
...
-  # by checking the removeable flag.
-  # There might be a better way in GNU/Linux systems.
-  removable=`lsblk -d $device --output RM | ignore_first_line | trim`
-
-  if [[ "$removable" == "1" ]]; then
+  eval "`udevadm info --query=property --export --export-prefix=UDEV_ --name=$disk`"
+  if [[ "`lsblk -d $device --output RM | ignore_first_line | trim`" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_MEDIA_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_BUS" == "usb" ]]
```
Comment (agherzan) [inline scripts/linux.sh]: So you return False when is removable?

File: scripts/linux.sh (hunk 1 above)
Comment (jviotti) [inline scripts/linux.sh]: What do you mean by error? I'm logging a property here. The output from this script is then parsed as a JavaScript object.

File: scripts/linux.sh (hunk 2 above)
Comment (jviotti) [inline scripts/linux.sh]: I consider `system` drives to be non-removable drives. I got the "system" naming convention from somewhere else, I think another NodeJS module, but I see it causes some confusion. Maybe I can rename to `removable`?

Comment (agherzan) [inline scripts/linux.sh]: Uh. With this explanation this makes sense. You can either rename it or state this in a comment or so.

File: scripts/linux.sh (hunk 1 above)
Comment (agherzan) [inline scripts/linux.sh]: Got it. I though it was just pure logging. All good

File: scripts/linux.sh (hunk 2 above)
Comment (jviotti) [inline scripts/linux.sh]: Cool, I'm making a note to rename this property in a new pull request.

**Pull request merged**